documentation/release-notes/topics/version_3.rst

```diff
@@ -3,6 +3,107 @@
+
+It is turned *Off* by default.
+
+=============================
+New Feature: Sensor Dashboard
+=============================
+A new sensor dashboard has been added to allow users to better visualize collected sensor data for an asset. Users will be able to toggle between list and card view for sensors.
+
+===============================
+Application Layout Improvements
+===============================
+- The search navigation bar has been added to the top of the product to allow for better access to Quick and Advanced Search features
+- The Account Management link has been moved from the bottom left of the application layout to the top right
+
+================================================
+2D and 3D Layout: Enhanced Heatmap Visualization
+================================================
+Heatmap visualization has been enhanced to be more accurate and more performant.
+
+In addition, there was a host of bug fixes, usability and performance improvements.
+
+=================================
+Sensor Data Download Improvements
+=================================
+Sensor data export has been improved to allow for up to 180 days of raw or daily summary data. Provided that the range falls within license limits and there is corresponding collected data for the selected sensor. This feature is available from both the application GUI and the API.
+
+==========================
+Other notable improvements
+==========================
+- "Inventory" has been added as an asset lifecycle state with support throughout the product
+- Desired rack temperature property update bulk action has been extended to racks
+- Improvements have been done to the discovery and monitoring sub-systems to allow for better automatic detection and management of asset IP Address changes
+- The multi-rack view has been enhanced to allow for image or PDF export of side-by-side rack elevations
+- Data grids have been improved throughout the product with better column filtering, clear column filter buttons, responsive behavior, and paging enhancements
+- The asset status has been added to the assets by type and by location grids
+- The asset monitoring state has been added to the advanced search column selector
+- The asset hierarchy tree load time has been improved
+- The rack asset audit page has been improved with a set of UX improvements and bug fixes
+- The main navbar -> settings area has been re-organized to improve UX
+- The license page has been improved to improve UX
+
+============
+Known issues
+============
+- **AS-12726** Certain Rack PDUs (usually older models) may return unexpected data when an outlet control operation is performed, this may cause the work order to be marked as failed even if it was performed correctly. Standard troubleshooting techniques such as making sure the latest firmware is loaded and restarting the rack PDU controller may resolve the issue. if this fails, please contact Hyperview support.
```

Comment:
This is the first I am hearing about it.
I will continue a conversation in that ticket.
I would suggest removing this.

Reply:
Discussed.